Makefile

```diff
@@ -46,7 +46,7 @@ requirements: pip
 	- $(PYTHON) -m pip install -r requirements.txt
 
 check:
-	inspekt checkall --disable-lint W,R,C,E0203,E0601,E1002,E1101,E1102,E1103,E1120,F0401,I0011,E1003,W605 --disable-style W605,W606,E501,E265,W601,E402,E722,E741 --exclude avocado-libs,scripts/github --no-license-check
+	inspekt checkall --disable-lint W,R,C,E0203,E0601,E1002,E1101,E1102,E1103,E1120,F0401,I0011,E1003,W605,I1101 --disable-style W605,W606,E501,E265,W601,E402,E722,E741,E721 --exclude avocado-libs,scripts/github --no-license-check
 	pylint --errors-only --disable=all --enable=spelling --spelling-dict=en_US --spelling-private-dict-file=spell.ignore *
```

Comment:
For  E721 `do not compare types, for exact checks use is / is not, for instance checks use isinstance(), I think we can easily fix instead of disable. We already did the similar fix in tp-libvirt https://github.com/autotest/tp-libvirt/pull/5066.  What do you think?

Reply:
@dzhengfy , fixing E721 issues, and leave I1101 unchecked now